integration_tests/tests/test_tokenization.rs

```diff
@@ -546,3 +548,208 @@ async fn close_tokenized_attestation_success() {
     assert_eq!(token_account.mint, attestation_mint_pda);
     assert_eq!(token_account.amount, 0);
 }
+
+#[tokio::test]
+async fn update_tokenized_attestation_success() {
```

Comment:
name is not descriptive of the test?

Reply:
it's accurate since updating the tokenized attestation requires closing -> opening, but maybe not descriptive enough.